lib/matplotlib/image.py

```diff
@@ -160,6 +160,25 @@ def flush_images():
         flush_images()
 
 
+def _resample(
+        image_obj, data, out_shape, transform, *, resample=None, alpha=1):
```

Comment:
The docstring could use some short explanations what the parameters mean.

Reply:
documented some of them; left those which would basically just be "forwarded to _image.resample".